### ISSUE TITLE
config.ini not updating

### ISSUE THREAD

**Issue by daHaimi** [koala-framework/koala-framework#459]
Hi there,

i as switching from memcached to redis (I modified Kwf_Cache_Simple and added the Zend Backend for this) and changed my config this way:

```ini
;server.memcache.host = localhost
server.redis.sock = /var/run/redis/redis.sock
```

The Kwf_Cache_Simple modification looks like
```php
...
         } else if (Kwf_Config::getValue('server.memcache.host')) {
            $ret = 'memcache';
        } else if (Kwf_Config::getValue('server.redis.sock')) {
            $ret = 'redis';
        } ...
```

I tried almost everything (including full composer update, bootstrapping with clear-cache, --type=config), But when I call Kwf_Cache_Simple::add, I still get an error, that the MemcachePool is not available.

Is there any kind of other caching mechanism I did not find yet for the config files?
I use only one config (config.ini) file for simplicity. May the Error be found there?

**Comment by daHaimi** [comment]
Okay, I just found out that there was a spare config.local.ini on th server from the times I was deploying manually. I found this after finding this file in the static excludes in the kwf-deploy.

**Comment by nsams** [comment]
Feel free to send us a Pull Request for redis support...

**Comment by daHaimi** [comment]
I haven't worked with pull requests up to now, can I send you the diff? there are only changes in the Kwf_Cache and a new Kwf_Cache_Adapter_Redis.

**Comment by nsams** [comment]
It's not that complicated...

- fork the repo on github
- add the remote to your local repository
- create a branch with the commit and push it to your fork
- create a pull request in the github webinterface

https://yangsu.github.io/pull-request-tutorial/


...but you can also send me the patch if you prefer that.

**Comment by daHaimi** [comment]
Okay, now I've found time to correct the "Quick & Dirty"-Solution I had before to match the Zend Framework Coding Guidelines. Since Patchfiles are not supported, I have attached it zipped.

After three weeks in use, I must say that Redis works faster, more stable and more relaiable even with huge amounts of data.

[kwf.patch.zip](https://github.com/koala-framework/koala-framework/files/68602/kwf.patch.zip)
